data.json

```diff
@@ -15615,9 +15615,9 @@
         "linux": {
             "root": "viking",
             "symlinks": [
+                "gpxsee",
```

Comment:
This one should be just after `gpx-viewer` and before `gpxviewer`
If you want to do it the lazy way, you can use this https://github.com/bil-elmoussaoui/numix-tools/blob/master/order_db.py just copy paste it in the numix-core repo and run it `python3 order_db.py`